tests/services/messenger/translators/front.spec.ts

```diff
@@ -1,10 +1,48 @@
 /// <reference types="mocha" />
 import { expect } from 'chai';
 
+import * as readFile from 'fs-readfile-promise';
```

Comment:
If you're just using it for reading a file:
```
import * as FS from 'fs';
const readFile = Promise.promisify(FS.readFile);
```
I'm all in favour of utilities, but this seems extremely heavy for just reading a file when an extra line will get you exactly the same results.

Maybe we should be including a utility service for general FS access.